src/main/java/core/basesyntax/dao/BetDao.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Bet;
+import java.util.List;
+
+public class BetDao implements Dao<Bet> {
```

Comment:
```suggestion
public class BetDaoImpl implements GenericDao<Bet> {
```

same for UserDaoImpl